src/sentry/api/endpoints/organization_transaction_anomaly_detection.py

```diff
@@ -0,0 +1,92 @@
+from datetime import datetime
+
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.bases import OrganizationEventsEndpointBase
+from sentry.api.utils import get_date_range_from_params
+from sentry.search.events.builder import QueryBuilder
+from sentry.snuba.dataset import Dataset
+
+
+class OrganizationTransactionAnomalyDetectionEndpoint(OrganizationEventsEndpointBase):
+    def get(self, request: Request, organization) -> Response:
+        start, end = get_date_range_from_params(request.GET)
+        query_start, query_end, granularity = self.map_snuba_queries(
```

Comment:
The (new) `query_start` and `query_end` should not introduce any timezone related nuances since `map_snuba_queries` is working with Unix timestamps but just wanted to double-check.